Sources/Tuplet.swift

```diff
@@ -405,7 +410,7 @@ public struct Tuplet: NoteCollection {
             notes[first[0]] = tuplet
         } else if first.count == 2 {
             // This is the same as the method that calls this. Should be able to be extracted into helper function
-            let tupletIndexes = Set(flatIndexes.map { $0[0] })
+            let tupletIndexes = Set(flatIndexes.map({ $0[0] })).sorted(by: { $0 > $1 })
```

Comment:
Do you need to do `sorted` or can you just do `reversed()`?

Reply:
I cannot just call `reversed()` because:
1. It is a Set
2. There is no claim that `flatIndexes` are sorted in ascending order.

Reply:
Ah yes. That makes sense. Thanks.